.github/ISSUE_TEMPLATE.MD

```diff
@@ -0,0 +1,21 @@
+**Do you want to request a *feature* or report a *bug*?**
```

Comment:
Should we structure our `_TEMPLATE` in a way that would allow us to directly copy-paste it as the issue template in the [project settings](https://github.com/CircleCI-Public/circleci-cli/settings)? So more like:
```
[ ] I have read [Contribution Guidelines](./CONTRIBUTING.md).
[ ] I have looked for similar issues reported previously and didn't find anything relevant.
[ ] The issue I'm reporting is not a security concern (if it is, it should be reported [here](https://circleci.com/security/)).

**Do you want to request a *feature* or report a *bug*?**

**What is the current behavior?**
 
Can you provide an example?
 
**What is the expected/desired behavior?**
  
**Which version of the CLI and OS are you using? Did this work in previous versions?**
 
Please provide the output of `circleci version` and `circleci diagnostic`.
```